doc/reference/std/misc/rtd.md

```diff
@@ -71,6 +71,34 @@ type object.
 ```
 :::
 
+## type=?
+``` scheme
+(type=? typ1 type2) -> bool
+
+  typ1 := type object
+  typ2 := other type object
+```
+
+Returns true if the two type objects have the same `type-id`.
+This is the preferred equality predicate for types.
+`eq?` and `eqv?` should also work, but at present
+`equal?` seems to be broken.
```

Comment:
this comment is misleading.

It doesn't seem to be broken, it simply doesn't work by necessity because `equal?` ignores the extra fields in gerbil type descriptors since all it sees is a the `##type-type` type and doesn't know how to process them

Reply:
Couldn't we tell `equal?` to look at the `type-id` field and none other?
Where is the hook for `equal?` and why can't we do the right thing with it?

Reply:
Unfortunately there is no equal hook to do the right thing.